tests/components/sensibo/test_select.py

```diff
@@ -154,87 +151,3 @@ async def test_select_set_option(
 
     state = hass.states.get("select.kitchen_light")
     assert state.state == STATE_UNAVAILABLE
-
-
-@pytest.mark.parametrize(
-    "load_platforms",
-    [[Platform.SELECT]],
-)
-async def test_deprecated_horizontal_swing_select(
-    hass: HomeAssistant,
-    load_platforms: list[Platform],
-    mock_client: MagicMock,
-    entity_registry: er.EntityRegistry,
-    issue_registry: ir.IssueRegistry,
-) -> None:
-    """Test the deprecated horizontal swing select entity."""
-
-    config_entry = MockConfigEntry(
-        domain=DOMAIN,
-        data=ENTRY_CONFIG,
-        entry_id="1",
-        unique_id="firstnamelastname",
-        version=2,
-    )
-
-    config_entry.add_to_hass(hass)
-
-    entity_registry.async_get_or_create(
-        SELECT_DOMAIN,
-        DOMAIN,
-        "ABC999111-horizontalSwing",
-        config_entry=config_entry,
-        disabled_by=None,
-        has_entity_name=True,
-        suggested_object_id="hallway_horizontal_swing",
-    )
-
-    with patch("homeassistant.components.sensibo.PLATFORMS", load_platforms):
-        await hass.config_entries.async_setup(config_entry.entry_id)
-        await hass.async_block_till_done()
-
-    state = hass.states.get("select.hallway_horizontal_swing")
-    assert state.state == "stopped"
-
-    # No issue created without automation or script
-    assert issue_registry.issues == {}
-
-    with (
-        patch("homeassistant.components.sensibo.PLATFORMS", load_platforms),
-        patch(
-            # Patch check for automation, that one exist
-            "homeassistant.components.sensibo.select.automations_with_entity",
-            return_value=["automation.test"],
-        ),
-    ):
-        await hass.config_entries.async_reload(config_entry.entry_id)
-        await hass.async_block_till_done(True)
-
-    # Issue is created when entity is enabled and automation/script exist
-    issue = issue_registry.async_get_issue(DOMAIN, "deprecated_entity_horizontalswing")
-    assert issue
-    assert issue.translation_key == "deprecated_entity_horizontalswing"
-    assert hass.states.get("select.hallway_horizontal_swing")
-    assert entity_registry.async_is_registered("select.hallway_horizontal_swing")
```

Comment:
Are there translation strings that can be removed?

Reply:
Always forgets those...